rest_framework/fields.py

```diff
@@ -59,7 +59,11 @@ def is_simple_callable(obj):
     if not (function or method):
         return False
 
-    args, _, _, defaults = inspect.getargspec(obj)
+    if six.PY2:
+        args, _, _, defaults = inspect.getargspec(obj)
+    else:
+        args, _, _, defaults = inspect.getfullargspec(obj)[:4]
+
```

Comment:
Great stuff. Let's move this into a `getargspec()` function in `compat.py`, then we'll be good to go.